assets/javascripts/discourse/components/explorer-schema/enum-info.js

```diff
@@ -0,0 +1,9 @@
+import Component from "@glimmer/component";
+
+export default class EnumInfo extends Component {
+  get enuminfo() {
+    return Object.entries(this.args.col.enum).map(([value, name]) => {
+      value, name;
+    });
```

Comment:
Whoops, I was missing parens:

```suggestion
    return Object.entries(this.args.col.enum).map(([value, name]) => ({
      value, name;
    }));
```